js/kucoin.js

```diff
@@ -1935,6 +1935,33 @@ module.exports = class kucoin extends Exchange {
         return this.parseTransactions (responseData, currency, since, limit, { 'type': 'withdrawal' });
     }
 
+    async parseBalance (response, params) {
```

Comment:
Making parseBalance async here is not entirely correct. All parsers are synchronous.